great_expectations/agent/actions/run_onboarding_data_assistant.py

```diff
@@ -0,0 +1,86 @@
+from typing import TYPE_CHECKING
+
+from great_expectations.agent.actions.action import (
+    ActionResult,
+    AgentAction,
+    CreatedResource,
+)
+from great_expectations.agent.models import RunOnboardingDataAssistantEvent
+from great_expectations.exceptions import StoreBackendError
+
+if TYPE_CHECKING:
+    from great_expectations.core import ExpectationSuite
+
+
+class RunOnboardingDataAssistantAction(AgentAction):
+    def run(self, event: RunOnboardingDataAssistantEvent, id: str) -> ActionResult:
+        expectation_suite_name = f"{event.data_asset_name} onboarding assistant suite"
+        checkpoint_name = f"{event.data_asset_name} onboarding assistant checkpoint"
+
+        # ensure resources we create don't already exist
+        try:
+            self._context.get_expectation_suite(
+                expectation_suite_name=expectation_suite_name
+            )
+            raise ValueError(
+                f"Onboarding Assistant Expectation Suite `{expectation_suite_name}` already exists. "
+                + "Please rename or delete suite and try again"
+            )
+        except StoreBackendError:
+            # resource is unique
+            pass
```

Comment:
Could we log here?

Reply:
sure, happy to update, but curious why? i'm not immediately seeing the benefit, since this is the successful path and we'll reach this most times the code runs.

Reply:
@joshua-stauffer agree, logging here would just be noisy